backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHBackend.scala

```diff
@@ -89,6 +92,19 @@ object CHBackendSettings extends BackendSettings with Logging {
   }
 
   override def utilizeShuffledHashJoinHint(): Boolean = true
+  override def supportShuffleWithProject(
+      outputPartitioning: Partitioning,
+      child: SparkPlan): Boolean = {
```

Comment:
the output of partial agg is different with the vanilla spark, for example, avg function, there are sum and count for the vanilla spark, but only avg for gluten + ch backend.

Reply:
I know this. In this case, we don't support and I have not found any case with expressions for `HashAggregateExec` at now.